extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -165,6 +187,40 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
       addExperimentIdToElement(verifierEid, this.element);
     }
 
+    const correlator = getCorrelator(this.win);
+    this.ampAnalyticsPageLoadMetricsConfig_ = dict({
+      'requests': {
+        'fvt': 'https://csi.gstatic.com/csi?s=a4a' +
+            `&c=${correlator}` +
+            '&firstVisibleTime.${firstVisibleTime}',
```

Comment:
I had the format wrong in my original requirements.  Please follow the format that Warren suggested:
```
https://csi.gstatic.com/csi?s=a4a&c=${PAGE_CORRELATOR}&met.a4a=firstVisibleTime.${firstVisibleTime}
```

Reply:
Fixed